python/checkout_steps.py

```diff
@@ -0,0 +1,55 @@
+# Import all the things
+import pandas as pd
+import requests
+
+# Retreive data from Redash - checkout steps completed by each trainee
+# Remove trainees who had anything happen before 2019-01-01 as we were not 
+# systematically tracking this back then
+
+checkout_progress_url = "http://redash.carpentries.org/api/queries/295/results.json?api_key=Wffe5EPpdkDhcbdX0NhtSOit1qKpfBrzqqtAKvWv"
```

Comment:
just to be safe, let's make this query private (I regenerated a new API key). The internal IDs to each individual should remain private as they carry a high risk of being cross-referenced with other datasets.